applications/OptimizationApplication/python_scripts/controls/thickness/shell_thickness_control.py

```diff
@@ -203,7 +203,7 @@ def Update(self, new_control_field: ContainerExpressionTypes) -> bool:
         if not IsSameContainerExpression(new_control_field, self.GetEmptyField()):
             raise RuntimeError(f"Updates for the required element container not found for control \"{self.GetName()}\". [ required model part name: {self.model_part.FullName()}, given model part name: {new_control_field.GetModelPart().FullName()} ]")
 
-        if KratosOA.ExpressionUtils.NormL2(self.control_field - new_control_field) > 1e-9:
+        if KratosOA.ExpressionUtils.NormL2(self.control_field - new_control_field) > 1e-15:
```

Comment:
Should we move this tolerance into JSON file to allow changing it?

Reply:
I do not know honestly, I am open to suggestions